src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1475,4 +1475,12 @@ Please specify one of them using the `--source` option to proceed.</value>
   <data name="PackageAlreadyInstalled" xml:space="preserve">
     <value>A package version is already installed. Installation cancelled.</value>
   </data>
+  <data name="EnableAdminSettingFailed" xml:space="preserve">
+    <value>Cannot enable %1. This setting is controlled by policy. For more information contact your system administrator.</value>

```

Comment:
>This setting is controlled by policy. [](http://example.com/codeflow?start=29&length=37)

nit: do you foressee other cases admin setting will fail to anable/disable? If yes, the wording may need to be revisited.